cli/ts/utils.ts

```diff
@@ -142,6 +142,21 @@ const isPathExist = (paths: Array<string>): [boolean, string] => {
     return [true, null]
 }
 
+const CompareOnchainValue = (
+    name: string,
+    onChainValue: any,
+    offChainValue: any,
+): boolean => {
+    if (onChainValue !== offChainValue) {
+        console.error(`Error: ${name} mismatch.`)
+        console.error("     onChainValue: " + onChainValue)
```

Comment:
Why the spacing on these logs?

Reply:
The spacing in these logs is intended to make the output more noticeable and easier to read.